package-lock.json

```diff
@@ -6505,8 +6465,28 @@
         "react-dom": "^16.8.0 || ^17.0.0-rc.1 || ^18.0.0 || ^19.0.0-rc.1"
       }
     },
+    "node_modules/@react-aria/collections": {
```

Comment:
these changes are in master, so once we update the parent branch and then this one the diff in the package lock should get smaller